kalite/topic_tools/__init__.py

```diff
@@ -261,44 +306,10 @@ def recurse_nodes(node):
     recurse_nodes(topictree)
 
     node_cache["Exercise"] = get_exercise_cache()
-    node_cache["Video"] = get_video_cache()
     node_cache["Content"] = get_content_cache()
 
     return node_cache
 
-
-def get_ancestor(node, ancestor_id, ancestor_type="Topic"):
-    potential_parents = get_node_cache(ancestor_type).get(ancestor_id)
-    if not potential_parents:
-        return None
-    elif len(potential_parents) == 1:
-        return potential_parents
-    else:
-        for pp in potential_parents:
-            if node["path"].startswith(pp["path"]):  # find parent by path
-                return pp
-        return None
-
-def get_parent(node, parent_type="Topic"):
-    return get_ancestor(node, ancestor_id=node["parent_id"], ancestor_type=parent_type)
-
-def get_videos(topic):
-    """Given a topic node, returns all video node children (non-recursively)"""
-    return filter(lambda node: node["kind"] == "Video", topic["children"])
-
-
-def get_exercises(topic):
```

Comment:
I remember writing some of these early on as helpers for manually traversing and doing stats on the topic tree data; as master burner, I leave it to your discretion, but some may be handy.